pwa/src/components/utrecht/topNav/TopNav.css

```diff
@@ -0,0 +1,6 @@
+.utrecht-navhtml > svg {
```

Comment:
When writing custom css on a utrecht component, we want to add a custom className. So instead of directly targeting `.utrecht-navhtml`, we can add our own class (e.g. `NavHtml`). This makes it easier to keep track of where we make changes.

Also, this makes it possible to make changes depending on our own context.